packages/mdc-textfield/foundation.ts

```diff
@@ -322,18 +293,17 @@ class MDCTextFieldFoundation extends MDCFoundation {
   }
 
   /**
-   * @return {boolean} If a custom validity is set, returns that value.
-   *     Otherwise, returns the result of native validity checks.
+   * Otherwise, returns the result of native validity checks.
```

Comment:
It looks like you accidentally nuked the first line of the parameter description

Reply:
Done